### DISCUSSION TITLE
Simplify adding sub-users to servers

### DISCUSSION THREAD

**Discussion by NekrosBurek** [pelican-dev/panel#122] [Ideas]
Currently adding new sub-users (sub-owners) to servers can be a pain.
First the option of me needing to enter users full email address instead of having a dropdown if the user/email is already a registered user as well also to have option to select all when using permissions and not just per group as i would then need to click trough all permissions when i want to give user all permissions

**Comment by TekExplorer** [comment]
> having a dropdown if the user/email is already a registered user

that could be fine for admins, but normal users should not have a way of getting the email addresses of people who use the panel just like that.

**Comment by lancepioch** [comment]
Thanks, we will be simplifying permissions, please follow: https://github.com/pelican-dev/panel/issues/193
